nucleus/cluster/cli/src/main/java/com/sun/enterprise/admin/cli/cluster/ChangeNodeMasterPasswordCommand.java

```diff
@@ -188,6 +189,22 @@ protected void createMasterPasswordFile() throws CommandException {
     }
 
 
+    protected void chmod(String args, File file) throws IOException {
```

Comment:
This is true.  The initial swipe at this was meant to only reach parity with how the master-password file is handled locally, but I'll follow up with some testing on an NIO2 implementation that can be used both locally and with node agents.  But first I want to implement it uniformly across both commands and test to make sure it actually works on multiple platforms.